BlogTemplate.Tests/Pages/EditTests.cs

```diff
@@ -5,15 +5,16 @@
 using BlogTemplate._1.Pages;
 using BlogTemplate._1.Services;
 using BlogTemplate._1.Tests.Fakes;
+using Microsoft.AspNetCore.Mvc.RazorPages;
 using Xunit;
 using static BlogTemplate._1.Pages.EditModel;
 
-namespace BlogTemplate.Tests.Pages
+namespace BlogTemplate._1.Tests.Pages
 {
-    class EditTests
+    public class EditTests
     {
         [Fact]
-        public void UpdatePost_TitleIsUpdated_UpdateSlug()
+        public void UpdatePostP2P_TitleIsUpdated_UpdateSlug()

```

Comment:
Expand the P2P.  You know what it means today (as you write it), but will you know what it means in 6 months?  Also the naming convention we've used is `<MethodName>_<scenario>_<expectation>`, so the P2P should be separated from the method name by at least and underscore.